From f3e2f40c332166713fc06732445574f1da2ab281 Mon Sep 17 00:00:00 2001
From: Siddharth Seth <sseth@apache.org>
Date: Mon, 19 Nov 2012 16:43:29 +0000
Subject: [PATCH 0599/1357] MAPREDUCE-4723 amendment. Fix JobHistory Event handling. (Contributed by Sandy Ryza)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1411292 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 74dce7dd5c124579e7a1bd7bf7ca7b085786f6fa)
---
 .../jobhistory/JobHistoryEventHandler.java         |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/jobhistory/JobHistoryEventHandler.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/jobhistory/JobHistoryEventHandler.java
index 9f0b906..489c3d2 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/jobhistory/JobHistoryEventHandler.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/jobhistory/JobHistoryEventHandler.java
@@ -601,7 +601,7 @@ public class JobHistoryEventHandler extends AbstractService
       setSummarySlotSeconds(summary, context.getJob(jobId).getAllCounters());
       break;
     default:
-      throw new YarnException("Invalid event type");
+      break;
     }
   }
 
-- 
1.7.0.4

